### PR TITLE
Add features

### PR DESCRIPTION
**Pull request by CarlosZiegler**
Hello @crisanlucid ,
here is my Pull request,
 - Create "Landing page"
 - add Basic and dirty Internationalization
 - Add Husk for pre commit, pre push and commit messages
 - adjus Vitest for testing in future
 - add Conventional Commits
 - add routes
 - add prettier and ES Lint
 - Add Fake page About
 
 
All parts need refinement :) 
Greate Project

### CODE REVIEW

**Comment by crisanlucid** [comment]
we should split the tasks in different commits and versions

File: src/components/BionicReaderPage/index.tsx [hunk 1]
```diff
@@ -43,29 +43,29 @@ const useTextProcessing = () => {
...
+    console.log('wait...')
+    setIsDisabled(true)
 
-    const listPrepText = processData();
-    setPretext("processing...");
+    const listPrepText = processData()
+    setPretext('processing...')
 
-    await timeout(2000);
+    await timeout(2000)
```
Comment (auryn31) [inline src/components/BionicReaderPage/index.tsx:62]: you should remove this. why would you wait for something, that does not need any time?

Comment (crisanlucid) [reply src/components/BionicReaderPage/index.tsx:62]: end-users like animations

**Comment by crisanlucid** [comment]
thank you

**Pull request merged**